PoseEstimation-TFLiteSwift/VideoCapture.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  VideoCapture.swift
+//  Awesome ML
+//
+//  Created by Eugene Bokhan on 3/13/18.
+//  Copyright © 2018 Eugene Bokhan. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+import CoreVideo
+
+public protocol VideoCaptureDelegate: class {
+    func videoCapture(_ capture: VideoCapture, didCaptureVideoFrame: CVPixelBuffer?, timestamp: CMTime)
+}
+
+public class VideoCapture: NSObject {
+    public var previewLayer: AVCaptureVideoPreviewLayer?
+    public weak var delegate: VideoCaptureDelegate?
+    public var fps = 30
+    
+    let captureSession = AVCaptureSession()
+    let videoOutput = AVCaptureVideoDataOutput()
+    let queue = DispatchQueue(label: "com.tucan9389.camera-queue")
+    
+    var lastTimestamp = CMTime()
+    
+    public func setUp(sessionPreset: AVCaptureSession.Preset = .vga640x480,
+                      cameraPosition: AVCaptureDevice.Position = .back,
+                      completion: @escaping (Bool) -> Void) {
+        self.setUpCamera(sessionPreset: sessionPreset,
+                         cameraPosition: cameraPosition,
+                         completion: { success in
+            completion(success)
+        })
+    }
+    
+    func setUpCamera(sessionPreset: AVCaptureSession.Preset, cameraPosition: AVCaptureDevice.Position, completion: @escaping (_ success: Bool) -> Void) {
```

Comment:
Many people think that 'completion' have to be called once.
But below code, completion() is omitted in else of guard.

Reply:
That's right. 
I fix to `fatalError()` rather than `completion()` because the developer should be able to catch early the error.